src/Services/ConsatImporter.php

```diff
@@ -31,7 +31,26 @@ public function import(string $dateStr, SinkFile $file): ConsatImporter
     {
         $this->importRecordCount = 0;
         $extractor = new ZipExtractor($file);
-        $mapFactory = new ConsatMapper($extractor->getDisk());
+        $extractor->extractContent();
+
+        // Load stop points first and prepare for NSR quay mapping.
+        $stopFilePath = sprintf('%s/StopPoint.csv', $extractor->getFullOutputDir());
+        $stopData = [];
+        if (($handle = fopen($stopFilePath, 'r')) !== false) {
+            $csvCols = fgetcsv($handle, 0, ';');
+            while (($values = fgetcsv($handle, 0, ';')) !== false) {
+                if (count($csvCols) !== count($values)) continue;
+                $csvRow = array_combine($csvCols, $values);
+                $stopData[$csvRow['Id']] = [
+                    'id' => $csvRow['ExternalId'],
+                    'name' => $csvRow['Name'],
+                ];
+            }
+            fclose($handle);
+        }
+
+        $mapFactory = new ConsatMapper($extractor->getDisk(), $stopData);
+        $mapFactory->except('StopPoint.csv');
```

Comment:
This should not be ripped out yet.

And if removed, a lot of stuff with it should go too, like an extra migration to remove this table, and removal of the mapping code for this in `ConsatMapper::mapStopPoint()`.

But we're not there yet (issue scope).